web/src/reducers/dataReducer.js

```diff
@@ -123,7 +123,11 @@ module.exports = (state = initialDataState, action) => {
         });
         if (!zone.exchange || !Object.keys(zone.exchange).length) {
           console.warn(`${key} is missing exchanges`);
+        } 
+        else if (!Object.keys(zone.production).length) {
+          zone.exchange = {};
```

Comment:
a comment would be welcome to understand why exchanges are changed due to empty production